src/inc-lib-init.php

```diff
@@ -1334,6 +1334,7 @@ function lovd_getVariantInfo ($sVariant, $sTranscriptID = '', $bCheckHGVS = fals
                  array('earliest_start',   'latest_start'),
                  array('earliest_end',     'latest_end'),
                  array('earliest_start',   'earliest_end'),
+                 array('latest_start',     'earliest_end'),
```

Comment:
This additional check makes sure that variants such as c.(1_3)_3del are now also being looked at.